src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -17,21 +17,31 @@ public class CommandResult {
     /** The application should exit. */
     private final boolean exit;
 
+    /** The application should start a review **/
+    private final boolean startReview;
+
+    /** The application should end a review **/
+    private final boolean endReview;
+
+
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
+    public CommandResult(
+            String feedbackToUser, boolean showHelp, boolean exit, boolean startReview, boolean endReview) {
```

Comment:
I would recommend reworking the parameters in here, it might get unnecessarily long in the future if we require other checks.